### ISSUE TITLE
Animated Custom Emojis PNGs (from Pleroma) show up as missing images

### ISSUE THREAD

**Issue by ashfurrow** [tootsuite/mastodon#13238]
This was discussed with the admin of `youdabomb.social` and myself, admin of `mastodon.technology`, here: https://mastodon.technology/@ashfurrow/103794534228017717

It looks like many of the emojis from the `youdabomb.social` domain have `nil` `image_remote_url` fields. This looks broken in Mastodon, with the characteristic "missing image" images:

<img width="326" alt="Screen Shot 2020-03-09 at 14 53 45" src="https://user-images.githubusercontent.com/498212/76247395-dd02b980-6215-11ea-9071-41f24bbbf55c.png">

I can run the following ActiveRecord query in the Rails console:

```rb
irb(main):011:0> CustomEmoji.where(domain: 'youdabomb.social').where(image_remote_url: nil).count
> 21
```

Let's examine the first as an example, and look at its original URL:

```rb
irb(main):013:0> CustomEmoji.where(domain: 'youdabomb.social').where(image_remote_url: nil)[5].uri
=> "https://youdabomb.social/emoji/blobs.gg/ablobglarezoom.png"
```

If we visit `https://youdabomb.social/emoji/blobs.gg/ablobglarezoom.png` then we'll see it's a PNG, but it's an _animate_ PNG. It turns out that _all_ the remote emojis that have missing images are animated PNGs. Some are also over the 50KB limit from Mastodon, but I've found examples of smaller animated PNGs too.

This is where I think the problem is. If I download the image and try to upload it to `mastodon.technology` as a local custom emoji, it fails with the error `can't be blank`. I tried looking in the server logs but didn't see anything obviously related.

I'm not sure about resolution here or next steps, but wanted to get it on the project's radar. Maybe we could expand custom emoji support on Mastodon to support animated PNGs? Or have a better experience for missing or unsupported remote emojis. 

I also found #10610 related, since my first instinct was to try to re-process the emoji (I worked around this by deleting one locally and asking the remote admin to send a toot with that same emoji). 

Thanks again! Let me know if I can provide more details.

**Comment by Gargron** [comment]
What surprises me is that a custom emoji record with no image attached exists. That's not supposed to happen. What should happen if an emoji file cannot be saved, just don't save the emoji, then it gets rendered as a shortcode, not a broken image. :confused:

**Comment by ashfurrow** [comment]
Right! That is surprising. The weird thing was that the image is _trying_ to load an asset from the mastodon.technology S3 bucket, but that asset doesn't exist. Please let me know if I can provide any more details at all.
